servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -163,6 +163,9 @@ quarkus.container-image.name=nessie
 %test.quarkus.security.users.embedded.users.test_user=test_user
 %test.quarkus.security.users.embedded.roles.admin_user=admin,user
 %test.quarkus.security.users.embedded.roles.test_user=test123
+%test.nessie.adapter.key-prefix=nessie-test
```

Comment:
I wonder, whether we should move all `test` profile properties into appropriate classes (not in this PR) after 1.0.0. WDYT?

Reply:
+1 #2240